adhocracy-plus/assets/js/app.js

```diff
@@ -8,6 +8,7 @@ import '../../../apps/dashboard/assets/ajax_modal.js'
 import '../../../apps/maps/assets/map-address.js'
 import '../../../apps/moderatorremark/assets/idea_remarks.js'
 import '../../../apps/newsletters/assets/dynamic_fields.js'
+import { initDashboardAccordeon } from '../../../apps/dashboard/helpers'
```

Comment:
Accordeon is spelt Accordion, sorry! I think it's also probably wrong in mB but just for code searches it's better to have it correct